src/main/java/parking/service/AdminService.java

```diff
@@ -164,7 +166,8 @@ public void detachParking(String username, HttpServletRequest httpRequest) throw
 
         accountRepository.detachParking(username, httpRequest);
         lotsRepository.removeParkingOwner(parkingLot.getNumber());
-
+        String userAgent = httpRequest.getHeader("User-Agent");
+        logRepository.insertActionLog(ActionType.DETACH_PARKING, accountRepository.findByUsername(username), parkingLot.getNumber(), null, null, null, userService.getCurrentUser(httpRequest), userAgent);
```

Comment:
Use the same coding style. I mean in one class you create a local variable with assigned account object, but in another, you just call a method and pass to the method as a parameter.